crates/core/src/model/scalar/datetime.rs

```diff
@@ -1,12 +1,8 @@
-use std::fmt;
-use std::ops;
```

Comment:
Side note: I prefer not to collapse `use`s because it increases the risk of conflicts.  Just a side note because its not worth going back and fixing.

(in general, its good to keep existing formatting when going into code)

Reply:
Yah sorry, that was my OCD kicking in.